### DISCUSSION TITLE
OSSP 2021 - Improving the OpenMS code quality

### DISCUSSION THREAD

**Discussion by OSSome01** [OpenMS/OpenMS#5366] [Code]
Hello @timosachsenberg @jpfeuffer,
I am interested in contributing towards improving the OpenMS code quality after its shift to C17. I saw that a project has been created in the repo for the same but it is empty, thus I created this issue for discussion. It would be great if I could get to know more about the project.

**Comment by timosachsenberg** [comment]
@OSSome01 hi if you want you can join our gitter channel https://gitter.im/OpenMS/OpenMS and we can have a more interactive chat there if you want

**Reply by timosachsenberg** [reply]
But in principle we would meet e.g., on a weekly basis to discuss improvements to the code base.
First focusing on the address sanitizer warnings and compile warnings. Then we can try to write existing code more concise using  structure bindings.
